action.yml

```diff
@@ -8,6 +8,10 @@ inputs:
   GITHUB_TOKEN:
     description: 'GitHub token'
     required: true
+  dry:
+    description: Just runs eslint, doesn't commit any changes
+    required: false
+    default: false
```

Comment:
Will this get cast as a string causing https://github.com/getsentry/action-eslint-fix/pull/7/files#diff-8cfead41d88ad47d44509a8ab0a109adR68 to always be true?